src/main/java/com/moodysalem/phantomjs/wrapper/CommandLineArgument.java

```diff
@@ -25,6 +25,10 @@ public CommandLineArgument(String template, String key, Object val) {
         objMap.put(key, val);
         return objMap;
     }
+    
+    public static String wrapTemplateName(String templateName){
```

Comment:
@GuillaumeC91 Bit of a confusing method name-maybe wrapCommandLineArgumentName or something